src/item.cpp

```diff
@@ -3373,7 +3374,34 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
 {
     std::stringstream ret;
 
+    int dirt_level = static_cast<int>( get_var( "dirt", 0 ) / 2000 );
```

Comment:
```suggestion
    int dirt_level = get_var( "dirt", 0 ) / 2000;
```
`get_var( "dirt", 0 )` is already an int, no need to cast it again.